zebrad/tests/acceptance.rs

```diff
@@ -1339,109 +1349,120 @@ fn lightwalletd_integration_test(test_type: LightwalletdTestType) -> Result<()>
         zebrad.expect_stdout_line_matches("loaded Zebra state cache .*tip.*=.*None")?;
     }
 
-    // Wait until `zebrad` has opened the RPC endpoint
-    zebrad.expect_stdout_line_matches(regex::escape(
-        format!("Opened RPC endpoint at {}", config.rpc.listen_addr.unwrap()).as_str(),
-    ))?;
-
-    // Launch lightwalletd
+    // Launch lightwalletd, if needed
+    let mut lightwalletd = if test_type.launches_lightwalletd() {
```

Comment:
This PR makes the `lightwalletd` launch optional in this test.

There's a lot of code movement in this test, you might be better reviewing it as a unified diff, or with `git --color-moved`.